src/components/ListItem.jsx

```diff
@@ -35,6 +43,10 @@ export function ListItem({ name, data, listToken }) {
 				<label htmlFor="listItem" className="ListItem-label">
 					{name}
 				</label>
+				<button type="submit" onClick={handleDeleteItem}>
+					{' '}
```

Comment:
@tinazarb @louieknolle 
Is `{' '}` needed?

Reply:
Good catch! Removed